src/Orckestra.Composer.Cart/Api/OrderController.cs

```diff
@@ -170,5 +178,47 @@ public virtual async Task<IHttpActionResult> GetGuestOrderByNumber(GetGuestOrder
 
             return Ok(viewModel);
         }
+
+        /// <summary>
+        /// Set an order in edit mode
+        /// </summary>
+        /// <param name="param">Parameters container</param>
+        [HttpPost]
+        [ActionName("edit-order")]
+        [ValidateModelState]
+        public virtual async Task<IHttpActionResult> EditOrder(Guid id)
+        {
+            if (id == Guid.Empty) return BadRequest($"{nameof(id)} cannot be empty");
+
+            var vm = await OrderService.CreateEditOrder(id).ConfigureAwait(false);
+
+            ComposerContext.EditingOrderId = id;
```

Comment:
inside ViewService save value ComposerContext.EditingOrderId - but save here cartDraft.CartName - it will contain OrderId but formated.
After we can use this in GetCart API and CartName = ComposerContext.EditingOrderId